lib/user.js

```diff
@@ -1191,9 +1214,13 @@ module.exports = function (config, userDB, couchAuthDB, mailer, emitter) {
 
   this.removeExpiredKeys = dbAuth.removeExpiredKeys;
 
-  this.confirmSession = session.confirmToken;
```

Comment:
Hello, you should take the PR 
https://github.com/colinskow/superlogin/pull/60/commits/08c8aacc17652c8b30f464d2434f22bf997744d8

instead of the last one because it has issues with the last changes.